xml/System.Reflection.Metadata/ImageFormatLimitationException.xml

```diff
@@ -17,7 +17,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
+    <summary>The exception that is thrown when the heap size limit is exceeded.</summary>
```

Comment:
I'd say more generally, that the exception is thrown when an attempt to write metadata exceeds a limit given by the format specification. Heap size limit is one of the limits.

Reply:
Ah great, thanks for the clarification. Does this sound good?:

```suggestion
    <summary>The exception that is thrown when an attempt to write metadata exceeds a limit given by the format specification. For example, when the heap size limit is exceeded.</summary>
```

Reply:
I can remove the example at the end if you don't feel like it should be included.

Reply:
@tmat I'll merge the above suggestion based on your comment, but feel free to let us know if you'd like us to change it.